alacritty/src/display/mod.rs

```diff
@@ -666,14 +665,11 @@ impl Display {
         self.size_info = new_size;
     }
 
+    // NOTE: Renderer updates are split off, since platforms like Wayland require resize and other
+    // OpenGL operations to be performed right before rendering. Otherwise they could lock the
+    // back buffer and render with the previous state. This also solves flickering during resizes.
+    //
     /// Update the state of the renderer.
```

Comment:
Split this into clippy fixes.

Reply:
I'll squash the clippy fixes with the rest, I don't think it should be relevant.

I just made it separate commits to make the review a little easier.

Reply:
Ok.